src/stdpar/plugin-PixelTriplets/CAHitNtupletGeneratorOnGPU.cc

```diff
@@ -81,45 +81,24 @@ CAHitNtupletGeneratorOnGPU::CAHitNtupletGeneratorOnGPU(edm::ProductRegistry& reg
          "h4",
          "h5");
 #endif
-
-  if (m_params.onGPU_) {
-#ifdef CUDAUVM_MANAGED_TEMPORARY
-    cudaCheck(cudaMallocManaged(&m_counters, sizeof(Counters)));
-#else
-    cudaCheck(cudaMalloc(&m_counters, sizeof(Counters)));
-#endif
-    cudaCheck(cudaMemset(m_counters, 0, sizeof(Counters)));
-  } else {
-    m_counters = new Counters();
-    memset(m_counters, 0, sizeof(Counters));
```

Comment:
This memset has been replaced by in-class initializer of ```Counters``` which should both have similar cost.